features/hydra/error.feature

```diff
@@ -103,23 +103,38 @@ Feature: Error handling
       And the JSON node "hydra:description" should be equal to "Update is not allowed for this operation."
       And the JSON node "trace" should exist
 
-    Scenario: Get an error during update of an existing relation with a non-allowed update operation
+    @dropSchema
+    @createSchema
+    Scenario: Do not get an error during update of an existing relation with a non-allowed update operation
+      When I add "Content-Type" header equal to "application/ld+json"
+      Given I send a "POST" request to "/related_dummies" with body:
+      """
+      {
+          "@type": "https://schema.org/Product",
+          "symfony": "laravel"
+      }
+      """
+      Then the response status code should be 201
+      And the response should be in JSON
+      And the JSON node "@id" should be equal to "/related_dummies/1"
+      And the JSON node "symfony" should be equal to "laravel"
+
       When I add "Content-Type" header equal to "application/ld+json"
```

Comment:
It should be split in several scenarios (a scenario should not have several `When` steps).

Reply:
Well the first lines are to check that the insertion is correct (mainly that the `@id` is `/related_dummies/1`). I can change the line 109 from `When` to `Given`.

Reply:
Yes it should follow the format:

```
Given ...
And ...
When ...
And ...
```

You can also split it in two scenarios.
Then

Reply:
@dunglas done